### ISSUE TITLE
UI elements in npm?

### ISSUE THREAD

**Issue by nicknikolov** [substack/browserify-handbook#51]
I really like the section about creating a small reusable widget in HTML. I wonder if people do that in practice and if one can find such packages in npm? @substack what's your experience? Do you write everything yourself everytime?

**Comment by mantoni** [comment]
I just started a project that uses HTML widgets in node modules.
Maybe this helps you to get started: https://github.com/geo-tales/geo-tales-mobile

**Comment by nicknikolov** [comment]
Thanks! Will use this as reference. Did you model it after the handbook?

**Comment by mantoni** [comment]
No. But I use browserify for all my frontend development at work. Unfortunately, that's not open source.
